@rhssk please fix the commit message, this line is too long:
```
cc1plus: warning: command-line option '-Wno-pointer-sign' is valid for C/ObjC but not for C++
```

> 5: UC4 Line exceeds max length (93>72): "cc1plus: warning: command-line option '-Wno-pointer-sign' is valid for C/ObjC but not for C++"
> ##[error]Process completed with exit code 1.

--- a/cmake/compiler/gcc/target_warnings.cmake
+++ b/cmake/compiler/gcc/target_warnings.cmake
@@ -114,4 +114,5 @@ endmacro()
 list(APPEND CXX_EXCLUDED_OPTIONS
   -Werror=implicit-int
   -Wold-style-definition
+  -Wno-pointer-sign
   )

